controls/sshd_spec.rb

```diff
@@ -276,10 +276,10 @@
 
 control 'sshd-26' do
   impact 1.0
-  title 'Server: Disable PAM'
-  desc 'Avoid challenge-response and password-based authentications.'
+  title 'Server: Disable challenge-response authentication'
+  desc 'Avoid challenge-response authentication.'
   describe sshd_config do
-    its('UsePAM') { should eq('no') }
```

Comment:
Do we have reasons to do that? In my eyes if we check for `UsePAM yes`, it gets to the hardening requirement, but its not a hardening requirement (at least anymore). Do I miss something?